chainerx_build_helper.py

```diff
@@ -56,7 +56,7 @@ def build_extension(self, ext):
             os.path.dirname(self.get_ext_fullpath(ext.name)))
         cmake_args = [
             '-DCHAINERX_BUILD_PYTHON=1',
-            '-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
+            '-DCMAKE_INSTALL_PREFIX=' + extdir,
```

Comment:
Installing to `extdir` is fine for you purpose? I mean, is it okay that it's not configurable?

Reply:
When I need to make this configurable I'll run cmake without `pip` so this is what I expect